pkg/frontend/openshiftcluster_preflightvalidation.go

```diff
@@ -0,0 +1,109 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+// NOTE: Make sure to change and always return 200, with any errors inside of the response message
+func (f *frontend) preflightValidation(w http.ResponseWriter, r *http.Request) {
```

Comment:
Would be helpful to add a comment describing what a `preflightValidation` request URL looks. [Example in ARO-RP frontend.](https://github.com/Azure/ARO-RP/blob/30370deecd647c9e70662f2c302299407eebfce1/pkg/frontend/admin_openshiftcluster_vmsizelist.go#L22)

Reply:
Added url as mentioned